PURL-TYPES.rst

```diff
@@ -317,6 +317,7 @@ including container images built by Docker and others:
 - Optional qualifiers may include:
 
   - ``arch``: key for a package architecture, when relevant
+  - ``os``: key for package os, when applicable
```

Comment:
```suggestion
  - ``os``: key for package operating system (os) with value such as ...
```
Also could explain how is this value discovered?

Reply:
Changed it to:

`key for package operating system (os) with value such as linux, darwin, windows, etc.`